src/main/java/games/strategy/engine/data/PseudoElement.java

```diff
@@ -0,0 +1,41 @@
+package games.strategy.engine.data;
+
+import org.xml.sax.Attributes;
+
+import com.google.common.base.Joiner;
+
+public class PseudoElement {
```

Comment:
This class is only used by `TripleaHandler`.  At a minimum, it should be package-private, but it could work also as a private nested class of `TripleaHandler`.